app/views/posts/index.html.erb

```diff
@@ -1 +1,38 @@
-<h1>Here is a list of all posts for a given user</h1>
\ No newline at end of file
+<div>
+  <div class= "show-wrapper">
+    <img src= <%= @user.photo %>  class= "show-image">
+    <div class= "show-details">
+      <h2 class='user-name'><%=@user.name%></h2>
+      <p class='post-count'>Number of posts: <%= @user.posts_counter %> </p>
+    </div>
+  </div>
+  <div class= "cards">
+    <%@posts.each do |post| %>
+      <div class= "post-card">
+        <div>
+          <h3>Post #<%= post.id%> </h3>
+          <%# <p><%= post.title%> </p> 
```

Comment:
- [ ] Great job so far, Kindly remove the commented code, commented code makes the code base hard to read.